.travis.yml

```diff
@@ -4,5 +4,18 @@ xcode_scheme: JenkinsiOSTests
 osx_image: xcode8.2
 xcode_sdk: iphonesimulator10.0
 
+cache:
+- bundler
+
+before_install:
+- bundle install
+
+branches:
+  only:
+    - master
+    - /^(\d+\.)?(\d+\.)?\d+(-\w+)?$/
+    - dev
+    - JENIOS-29/fastlane
```

Comment:
Shouldn't need this one - it will start building as soon as you open a pull request where the target branch is one of: 
```
- master
- /^(\d+\.)?(\d+\.)?\d+(-\w+)?$/
- dev
```